tests/upgrade/upgrade_test.go

```diff
@@ -61,12 +63,20 @@ var _ = ginkgo.Describe("[Upgrade]", func() {
 			cChainGenesisStr := network.Genesis.CChainGenesis
 			require.NoError(json.Unmarshal([]byte(cChainGenesisStr), cChainGenesis))
 			unscheduledActivationTime := uint64(upgrade.UnscheduledActivationTime.Unix())
-			cChainGenesis.Config.EtnaTime = &unscheduledActivationTime
+			cChainGenesis.Config.EtnaTimestamp = &unscheduledActivationTime
 			cChainGenesisBytes, err := json.Marshal(cChainGenesis)
 			require.NoError(err)
 			network.Genesis.CChainGenesis = string(cChainGenesisBytes)
 		}
 
+		// Previous version does not have unactivated upgrades.
+		// The default upgrade config usually sets the latest upgrade to be activated
+		// Use unscheduled
+		upgradeJSON, err := json.Marshal(upgrade.LatestUnscheduled)
+		require.NoError(err)
+		upgradeBase64 := base64.StdEncoding.EncodeToString(upgradeJSON)
+		network.DefaultFlags[config.UpgradeFileContentKey] = upgradeBase64
```

Comment:
I'm not sure why the upgrade json is needed (it is not parsed in coreth, afaik)

Reply:
This is the new network upgrades flag (not the `upgrade.json` under chain-config-dir`). I agree this is slightly confusing.